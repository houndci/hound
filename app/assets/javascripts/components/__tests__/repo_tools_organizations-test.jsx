import RepoToolsOrganizations from '../repo_tools_organizations.jsx';

it('renders a dropdown list containing organizations', () => {
  const organizations = [
    { id: 1, name: "Test org" }
  ]

  const wrapper = shallow(
    <RepoToolsOrganizations
      organizations={organizations}
    />
  );

  expect(wrapper).toMatchSnapshot();
});
