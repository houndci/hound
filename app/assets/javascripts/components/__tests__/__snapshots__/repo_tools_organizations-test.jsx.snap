exports[`test renders a dropdown list containing organizations 1`] = `
<div
  className="repo-tools-org dropdown">
  <i
    className="fa fa-caret-down" />
  <select
    className="quick-jump">
    <option
      disabled={true}>
      Quick jump to organization
    </option>
    <option
      value="div[data-org-name=\'Test org\']">
      Test org
    </option>
  </select>
</div>
`;
